<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_banque</class>
 <widget class="QDialog" name="dlg_banque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>124</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="Banqueframe">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>350</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>100</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(150,150,150);
border-radius: 5px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="UpSmallButton" name="AnnulModifupSmallButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>65</y>
      <width>34</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Images/images.qrc">
      <normaloff>:/Supprime.png</normaloff>:/Supprime.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="UpSmallButton" name="OKModifupSmallButton">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>65</y>
      <width>34</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Images/images.qrc">
      <normaloff>:/start.png</normaloff>:/start.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="NomBanquelabel">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>15</y>
      <width>29</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px</string>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="NomAbregelabel">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px</string>
    </property>
    <property name="text">
     <string>Abréviation
(10 caractères maxi)</string>
    </property>
   </widget>
   <widget class="UpLineEdit" name="NomAbregeupLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>36</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="UpLineEdit" name="NomBanqueupLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpLineEdit</class>
   <extends>QLineEdit</extends>
   <header>uplineedit.h</header>
  </customwidget>
  <customwidget>
   <class>UpSmallButton</class>
   <extends>QPushButton</extends>
   <header>upsmallbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>NomAbregeupLineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
